packages/ai/src/LLMClient/AnthropicProvider.ts

```diff
@@ -0,0 +1,97 @@
+import Anthropic from "@anthropic-ai/sdk";
```

Comment:
Reasonable move, to split this up.

Reply:
That said, this does make it tricky to determine what code was added / changed, of this part.

Reply:
IIRC only `LLMError` class is new, everything else in `LLMClient/` is old.

And even that class is used only for `timeout` errors (specific LLM providers don't try to normalize vendor lib errors to `LLMError` instances, which works because everything that's not an `LLMError` is treated as critical, which is what we want, for now).